eZ/Publish/API/Repository/ContentService.php

```diff
@@ -266,7 +266,7 @@ public function publishVersion(VersionInfo $versionInfo);
      * Removes the given version.
      *
      * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException if the version is in
-     *         published state or is a last version of the Content
+     *         published state or is a last version of the Content and not a draft
```

Comment:
`is a last version of the Content and not a draft`  => `is a last version of Content in published state`

Reply:
Updated